### ISSUE TITLE
 TypeError: Cannot read property 'emit' of undefined

### ISSUE THREAD

**Issue by jakejdavis** [fullcalendar/fullcalendar-angular#175]
Using configuration:
`  <full-calendar class="calendar" schedulerLicenseKey="" [plugins]="calendarPlugins"
    defaultView="timelineMonth"
    scrollTime= '00:00'
    allDayDefault="true"
    weekNumbers="true"
    weekNumberTitle= 'Wk'
    firstDay= 1
    slotWidth= 45
    timezone= 'local'
    locale= 'en-gb'
    [resources]="Config.url + 'api/calendar/resources/get'"
    [events]="Config.url + 'api/calendar/events/get'">
  </full-calendar>`

The error:
```
CalendarComponent.html:2 ERROR TypeError: Cannot read property 'emit' of undefined
    at Calendar.<anonymous> (fullcalendar-angular.js:234)
    at Calendar.publiclyTrigger (main.js:7078)
    at Calendar.dispatch (main.js:6765)
    at main.js:6733
    at Calendar.batchRendering (main.js:6823)
    at Calendar.hydrate (main.js:6730)
    at new Calendar (main.js:6632)
    at FullCalendarComponent.push../node_modules/@fullcalendar/angular/fesm5/fullcalendar-angular.js.FullCalendarComponent.ngAfterViewInit (fullcalendar-angular.js:211)
    at callProviderLifecycles (core.js:22406)
    at callElementProvidersLifecycles (core.js:22380)
```
is returned, which causes an error when parsing the JSON.

The error is caused by trying to emit info for a 'loading' property (which exists in EVENT_NAMES) where a loading EventEmitter is not declared.

**Comment by adorep** [comment]
This also breaks the calendar if you are using a function to load events like described here: https://fullcalendar.io/docs/events-function

I am currently avoiding the exception using this simple "workaround" (but i also don't need the loading event): `FullCalendarComponent.prototype['loading'] = { emit: _ => { } };`

**Comment by arshaw** [comment]
this is fixed in v4.1.0
